roles/kubernetes-apps/ansible/templates/nodelocaldns-config.yml.j2

```diff
@@ -0,0 +1,55 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nodelocaldns
+  namespace: kube-system
+  labels:
+    addonmanager.kubernetes.io/mode: EnsureExists
+
+data:
+  Corefile: |
+    {{ dns_domain }}:53 {
+        errors
+        cache 30
+        reload
+        loop
+        bind {{ local_dns_ip }}
+        forward . {{ clusterIP }} {
+                force_tcp
+        }
+        prometheus :9253
+        health {{ local_dns_ip }}:8080
+        }
+    in-addr.arpa:53 {
+        errors
+        cache 30
+        reload
+        loop
+        bind {{ local_dns_ip }}
+        forward . {{ clusterIP }} {
+                force_tcp
+        }
+        prometheus :9253
+        }
+    ip6.arpa:53 {
+        errors
+        cache 30
+        reload
+        loop
+        bind {{ local_dns_ip }}
+        forward . {{ clusterIP }} {
+                force_tcp
+        }
+        prometheus :9253
+        }
+    .:53 {
+        errors
+        cache 30
+        reload
+        loop
+        bind {{ local_dns_ip }}
+        forward . /etc/resolv.conf {
```

Comment:
https://github.com/kubernetes-sigs/kubespray/blob/master/roles/kubernetes-apps/ansible/templates/coredns-config.yml.j2#L16
This is to prevent DNS loops.

Reply:
Will need some help with this.
It's the first time I open a corefile and I would like some advice on how to incorporate your fix into the code.

Reply:
```
{% if resolvconf_mode == 'host_resolvconf' and upstream_dns_servers is defined and upstream_dns_servers|length > 0 %}
          forward . {{ upstream_dns_servers|join(' ') }} {
{% else %}
          forward . /etc/resolv.conf {
{% endif %}
```